packages/account/src/Sections/Security/Passwords/deriv-password.jsx

```diff
@@ -34,32 +34,49 @@ const DerivPassword = ({
     const platform_name_derivez = getPlatformSettings('derivez').name;
 
     const PlatformDescription = () => {
```

Comment:
could you please move this to a separate file?

Reply:
can you advice why do we need that? This is not reused else.

Reply:
Nesting components leads to recreation of components during each render.. this is creating unexpected bugs. for example, i had to refactor few components to fix an issue related to card flipping(when working in jurisdiction component), and in the card i am currently working on, i assume this could be a cause of the issue. so thought it would be better if we keep components separate.
https://react.dev/learn/preserving-and-resetting-state#different-components-at-the-same-position-reset-state
![Screenshot 2023-07-28 at 11 29 08 AM](https://github.com/binary-com/deriv-app/assets/122449658/535251bd-4946-41cd-973d-d945ed5fda2c)